pom.xml

```diff
@@ -86,7 +86,7 @@
     <jackson.datatype.groupId>com.fasterxml.jackson.datatype</jackson.datatype.groupId>
     <jackson.package>com.fasterxml.jackson</jackson.package>
     <!-- To upgrade jackson, check the jdk versions inside the jar and include any new versions in the shading in parquet-jackson. -->
-    <jackson.version>2.18.2</jackson.version>
+    <jackson.version>2.18.3</jackson.version>
```

Comment:
Should we manually change `jackson-databind.version` to use the same version? @Fokko

Reply:
Yes, I think this would be best :)